schemas/value_sources.json

```diff
@@ -199,5 +209,28 @@
     },
     "required": ["identifier", "source", "selector"],
     "additionalProperties": false
+  },
+  "supplementary_data_value_source": {
+    "description": "A reference to a value from supplementary data. When resolved this can be a number, string, array or null.",
+    "type": "object",
+    "properties": {
+      "identifier": {
+        "$ref": "common_definitions.json#/non_empty_string",
+        "description": "The top level key within `data` for the source of the data"
+      },
+      "source": {
+        "type": "string",
+        "enum": ["supplementary_data"]
```

Comment:
Non blocker. existing pattern but we should start using `const` when there is only one value. `const` didn't exist back in the day. Maybe a separate card to enforce across the board.

Reply:
Fixed this one and raised a card for the others 👍

Reply:
Nice one.